docs/resources/security_policy.md

```diff
@@ -48,42 +48,170 @@ resource "satori_security_policy" "security_policy" {
           }
         }
         action {
-          type = "APPLY_MASKING_PROFILE"
-          masking_profile_id = satori_masking_profile.masking_profile.id
+          type               = "APPLY_MASKING_PROFILE"
+          masking_profile_id = satori_masking_profile.masking_profile.id # as reference of previously created masking profile
         }
       }
       rule {
-        id = "2"
+        id          = "2"
         description = "rule 3"
-        active = true
+        active      = true
         criteria {
           condition = "IS_NOT"
           identity {
-            type = "GROUP"
+            type     = "GROUP"
             group_id = satori_directory_group.group2.id
           }
         }
         action {
-          type = "APPLY_MASKING_PROFILE"
+          type               = "APPLY_MASKING_PROFILE"
           masking_profile_id = satori_masking_profile.masking_profile.id
         }
       }
       rule {
-        id = "4"
+        id          = "4"
         description = "rule 4"
-        active = true
+        active      = true
         criteria {
           condition = "IS_NOT"
           identity {
-            type = "GROUP"
+            type     = "GROUP"
             group_id = satori_directory_group.group3.id
           }
         }
         action {
+          # type = "APPLY_MASKING_PROFILE"
           masking_profile_id = satori_masking_profile.masking_profile.id
         }
       }
     }
+    row_level_security {
+      active = false
+      rule {
+        id          = "rls1"
+        description = "rls 1 description"
+        active      = true
+        filter {
+          datastore_id = local.datastore_id
+          advanced     = false
+          logic_yaml   = <<EOT
+                          field:
+                            name: '33'
+                          filterName: Filter 1
```

Comment:
I think it better to use filter_1 name or "Filter 1" with " to avoid confuse with spaces